docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+
+Up next is a regex for whatever word you want to activate the bot to reply. I chose “please." As long as the user is polite, they’ll get a random soundtrack.
+
+The regex has “g” and “i” at the end so it ignores capitalization and checks globally for the word please. `playerOne` is a variable that can either be an empty array or will use `.match` to create an array with one element: the word “please”. `playerTwo` is a boolean that verifies whether the array `playerOne` has anything in it.
+
+The if statement requires that the tweet text contains the bot’s name and an array of at least one element was passed into `playerTwo`. If both of these come back as true, then we proceed to an area where the variable `replyText` is created, which includes a random element of the array, as well as the username of the person being replied to and a short message. `replyText` is passed into an object that contains two properties: `status` and `in_reply_to_status_id`. `status` is the actual text to be posted to the tweet, in our case the variable `replyText` is our status. `In_reply_to_status_id` is defined as `id`, which is a variable from the beginning of the `pressStart` function. `id` is a unique identifier of a tweet from Twitter’s API. This allows Twit to identify which tweet the bot will reply to as a comment. Finally, the else statement at the end will log a quote from Jurassic Park to the console if the user doesn’t say please, just so we can see why the bot didn't tweet back.
+
+```javascript
+	function pressSelect() {
+
+	    var weeklyReplyText = soundtrackArray[soundtrackArrayElement] + " Here's your soundtrack for the week!";
+	    T.post('statuses/update', { status: weeklyReplyText }, gameOver2);
+
+	    function gameOver2(err, reply) {
+	        if (err) {
+	            console.log(err.message);
+	            console.log("Game Over");
+	        } else {
+	            console.log('Tweeted: ' + reply.text);
+	        }
+	    }
+	}
+
+	 const job1 = schedule.scheduleJob(rule, pressSelect);
+
+	 job1.on("Every Day Tweet", pressSelect);
+```
+</br>
+
+Here is the function used to tweet on a schedule, which I’ve named `pressSelect`. `pressSelect` has the `replyText` variable, slightly changed to be an original tweet rather than a comment on a different tweet, but pulls from the same array. The `gameOver` function is also present, though renamed just to be safe. Since `gameOver` and `gameOver2` are local variables within their respective functions, there shouldn’t be any issues. However, I'm just being safe.
+
+The final part of this code is creating a variable called `job1`. `job1` is the `scheduleJob` method from the node-schedule object at the top of the code. I pass in `rule` created earlier in the code and `pressSelect` as parameters. We then use an event listener with `pressSelect` passed in again. That's it for the code.
+
```

Comment:
After the `Add the Twitter Bot Application Code` section, I wrote a new `Commit the Code and Push to GitHub` section:
https://deploy-preview-4492--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/nodejs-twitter-bot/#commit-the-code-and-push-to-github

I figured that if the project was starting as a repo on GitHub, then it would make sense to the user if they added the new bot code to the repo, and seem less like a dead-end in the instructions. Also, it lets the user perform a `git clone` on the Linode, as an alternative to the FileZilla instructions